src/battle-dex-search.ts

```diff
@@ -1177,6 +1177,8 @@ class BattleMoveSearch extends BattleTypedSearch<'move'> {
 			return ['technician', 'serenegrace'].includes(abilityid) || !moves.includes('powergem');
 		case 'aurawheel':
 			return species.baseSpecies === 'Morpeko';
+		case 'aurorabeam':
+			return !moves.includes('icebeam');
```

Comment:
> /ds aurorabeam, !icebeam:
> No Pokémon found.

```suggestion
```

I'd rather not.

Reply:
Xerneas only gets Ice Beam in Gen 7.

Reply:
I think you mean Aurora Beam.

Reply:
It got Aurora Beam in Gen 6... I'm not saying that it's viable though.

Reply:
Does Gen 7 Xerneas actually hit this code path, though? I suppose this is worth it if so.

Reply:
urkerab is saying Gen 6 Xerneas hits the path. That's why it's Xerneas tech in Gen 6.

Reply:
Works for me. Thanks for the changes!